test/library/input.spec.js

```diff
@@ -425,5 +425,40 @@ describe('NumberFormat as input', () => {
         shallow(<NumberFormat format="#### #### ####" mask={['D', 'D', 'M', '1', '2', 'Y', 'Y', 'Y']}/>)
       }).toThrow()
     })
+
+    it('should show the right decimal values based on the decimal scale provided', () =>{
```

Comment:
** Question **
Did this fail without the above change? I tested the behaviour (described it test) in codesandbox with current version and it did not fail.

Try removing your changes in source file and see if this fails.